packages/devextreme/js/__internal/viz/m_chart.ts

```diff
@@ -1209,77 +1256,89 @@ const dxChart = AdvancedChart.inherit({
 
       performActionOnAxes(allAxes, 'updateSize', panesCanvases);
 
-      const paneSizes = that.panes.reduce((sizes, pane) => {
+      const paneSizes = this.panes.reduce((sizes, pane) => {
         sizes[pane.name] = {
           height: pane.height,
           unit: pane.unit,
         };
         return sizes;
       }, {});
 
-      panesCanvases = shrinkCanvases(rotated, panesCanvases, paneSizes, getVerticalAxesMargins(verticalAxes), getHorizontalAxesMargins(horizontalAxes, getAxisMargins));
+      panesCanvases = shrinkCanvases(
+        rotated,
+        panesCanvases,
+        paneSizes,
+        getVerticalAxesMargins(verticalAxes),
+        getHorizontalAxesMargins(horizontalAxes, getAxisMargins),
+      );
       performActionOnAxes(allAxes, 'updateSize', panesCanvases);
       horizontalAxes.forEach(shiftAxis('top', 'bottom'));
       verticalAxes.forEach(shiftAxis('left', 'right'));
 
-      that.panes.forEach((pane) => _extend(pane.canvas, panesCanvases[pane.name]));
+      this.panes.forEach((pane) => _extend(pane.canvas, panesCanvases[pane.name]));
     }
   },
 
   _isArgumentAxisBeforeScrollBar() {
-    const that = this;
-    const argumentAxis = that.getArgumentAxis();
+    const argumentAxis = this.getArgumentAxis();
 
-    if (that._scrollBar) {
+    if (this._scrollBar) {
       const argAxisPosition = argumentAxis.getResolvedBoundaryPosition();
       const argAxisLabelPosition = argumentAxis.getOptions().label?.position;
-      const scrollBarPosition = that._scrollBar.getOptions().position;
+      const scrollBarPosition = this._scrollBar.getOptions().position;
 
-      return argumentAxis.hasNonBoundaryPosition() || scrollBarPosition === argAxisPosition && argAxisLabelPosition !== scrollBarPosition;
+      return argumentAxis.hasNonBoundaryPosition()
+      || (scrollBarPosition === argAxisPosition
+       && argAxisLabelPosition !== scrollBarPosition);
     }
 
     return false;
   },
 
   _getPanesParameters() {
-    const that = this;
-    const { panes } = that;
-    let i;
+    const { panes } = this;
     const params = [];
-    for (i = 0; i < panes.length; i++) {
-      if (that._getPaneBorderVisibility(i)) {
-        params.push({ coords: panes[i].borderCoords, clipRect: that._panesClipRects.fixed[i] } as never);
+    for (let i = 0; i < panes.length; i += 1) {
+      if (this._getPaneBorderVisibility(i)) {
+        params.push({
+          coords: panes[i].borderCoords,
+          clipRect: this._panesClipRects.fixed[i],
+        } as never);
```

Comment:
'never' isn't supposed to use this way. Ideally we are to introduce a new type for PanesParameters
```
interface PanesParameters {
  coords: someType
  clipRect: someType
}
```
and declare it like const params: PanesParameters[] = []; or at least params: object[] = [];
But i suppose this PR is all about refactoring, not conversion to TS

Reply:
Yes. we can add this type. But it leads to add types for 'coords' and 'clipRect'. Adding this types - continuous work. We can't add all types for all fields and objects.

Reply:
that's why i suggested to use  least params: object[] = [], it's better than using never here. But i suppose that's not yu code and this PR is enormous, so lets leave it as is )